mlrun/runtimes/pod.py

```diff
@@ -86,6 +87,15 @@
     "driver_security_context": sanitized_types["security_context"],
 }
 
+_AUTH_SECRET_PATTERN = re.compile(
+    re.escape(
```

Comment:
Would be best to start this with `^` to prevent situations where it's not the full name of the secret. For example, this will match `not-an-mlrun-auth-secrets.blablabla`.